api/src/main/java/org/openmrs/module/openhmis/inventory/web/ModuleWebConstants.java

```diff
@@ -69,5 +69,9 @@
 	public static final String LANDING_PAGE_EXTENSION_POINT_ID = "org.openmrs.module.openhmis.inventory.landing";
 	public static final String MANAGE_MODULE_PAGE_EXTENSION_POINT_ID = "org.openmrs.module.openhmis.inventory.manage.module";
 
+	public static final String MESSAGE_PROPERTIES_JS_URI = MODULE_ROOT + "inventoryMessageProperties.js";
+	public static final String MESSAGE_PAGE = MODULE_ROOT + "inventoryMessageProperties";
+	public static final String MESSAGE_PROPERTIES_JS = "/ws/module/openhmis/inventory/inventoryMessageProperties.js";
```

Comment:
Because the localheader.jsp file in the inventory includes html pages using defined strings from the module web constants.